src/Mustaches.ts

```diff
@@ -45,6 +45,11 @@ class Mustaches {
     return copiedFile
   }
 
+  async discovery({ source, data = {}, formatters = {} }: DiscoveryOptions) {
+    const doc = await this.readDoc(source)
+    return interpolate(doc, data, formatters)
```

Comment:
Also, I think a "pretty print" of discovery would be appreciated and more useful. Add it as an option maybe, to be able to use discovery in a web app for example.

Reply:
You can prettify using: `JSON.stringify(yourJson, null, 4)`

I don't like a method that either returns JSON or String, and I think the way you show it should be the user's choice